SoundWire support
SoundWire is a 2-Pin interface with data and clock line ratified
by the MIPI Alliance. SoundWire is used for transporting data
typically related to audio functions. SoundWire interface is
optimized to integrate audio devices in mobile or mobile inspired
systems. Say Y to enable this subsystem, N if you do not have such
a device
Depends: ACPI || OF
---
--- /var/tmp/portage/sys-kernel/gentoo-sources-5.19.17/image/usr/src/linux-5.19.17-gentoo//.config.orig	2022-10-31 13:57:59.705695527 -0400
+++ /var/tmp/portage/sys-kernel/gentoo-sources-5.19.17/image/usr/src/linux-5.19.17-gentoo//.config	2022-11-01 16:18:56.766610391 -0400
@@ -2272,8 +2272,6 @@
 CONFIG_REGMAP_SPI=m
 CONFIG_REGMAP_MMIO=y
 CONFIG_REGMAP_IRQ=y
-CONFIG_REGMAP_SOUNDWIRE=m
-CONFIG_REGMAP_SOUNDWIRE_MBQ=m
 CONFIG_REGMAP_SCCB=m
 CONFIG_REGMAP_SPI_AVMM=m
 CONFIG_DMA_SHARED_BUFFER=y
@@ -6697,7 +6695,6 @@
 CONFIG_SND_SOC_INTEL_SOF_DA7219_MAX98373_MACH=m
 CONFIG_SND_SOC_INTEL_SOF_SSP_AMP_MACH=m
 CONFIG_SND_SOC_INTEL_EHL_RT5660_MACH=m
-CONFIG_SND_SOC_INTEL_SOUNDWIRE_SOF_MACH=m
 # CONFIG_SND_SOC_MTK_BTCVSD is not set
 CONFIG_SND_SOC_SOF_TOPLEVEL=y
 CONFIG_SND_SOC_SOF_PCI_DEV=m
@@ -6739,7 +6736,6 @@
 CONFIG_SND_SOC_SOF_HDA=m
 CONFIG_SND_SOC_SOF_HDA_PROBES=m
 CONFIG_SND_SOC_SOF_INTEL_SOUNDWIRE_LINK_BASELINE=m
-CONFIG_SND_SOC_SOF_INTEL_SOUNDWIRE=m
 CONFIG_SND_SOC_SOF_XTENSA=m
 
 #
@@ -6837,7 +6833,6 @@
 CONFIG_SND_SOC_MAX98520=m
 CONFIG_SND_SOC_MAX98373=m
 CONFIG_SND_SOC_MAX98373_I2C=m
-CONFIG_SND_SOC_MAX98373_SDW=m
 CONFIG_SND_SOC_MAX98390=m
 CONFIG_SND_SOC_MAX98396=m
 # CONFIG_SND_SOC_MAX9860 is not set
@@ -6868,8 +6863,6 @@
 CONFIG_SND_SOC_RT1015=m
 CONFIG_SND_SOC_RT1015P=m
 CONFIG_SND_SOC_RT1308=m
-CONFIG_SND_SOC_RT1308_SDW=m
-CONFIG_SND_SOC_RT1316_SDW=m
 CONFIG_SND_SOC_RT5514=m
 CONFIG_SND_SOC_RT5514_SPI=m
 # CONFIG_SND_SOC_RT5616 is not set
@@ -6885,18 +6878,8 @@
 CONFIG_SND_SOC_RT5677_SPI=m
 CONFIG_SND_SOC_RT5682=m
 CONFIG_SND_SOC_RT5682_I2C=m
-CONFIG_SND_SOC_RT5682_SDW=m
 CONFIG_SND_SOC_RT5682S=m
-CONFIG_SND_SOC_RT700=m
-CONFIG_SND_SOC_RT700_SDW=m
-CONFIG_SND_SOC_RT711=m
-CONFIG_SND_SOC_RT711_SDW=m
-CONFIG_SND_SOC_RT711_SDCA_SDW=m
-CONFIG_SND_SOC_RT715=m
-CONFIG_SND_SOC_RT715_SDW=m
-CONFIG_SND_SOC_RT715_SDCA_SDW=m
 # CONFIG_SND_SOC_RT9120 is not set
-# CONFIG_SND_SOC_SDW_MOCKUP is not set
 # CONFIG_SND_SOC_SGTL5000 is not set
 CONFIG_SND_SOC_SIGMADSP=m
 CONFIG_SND_SOC_SIGMADSP_REGMAP=m
@@ -6937,7 +6920,6 @@
 CONFIG_SND_SOC_TSCS42XX=m
 # CONFIG_SND_SOC_TSCS454 is not set
 # CONFIG_SND_SOC_UDA1334 is not set
-# CONFIG_SND_SOC_WCD938X_SDW is not set
 CONFIG_SND_SOC_WM5102=m
 # CONFIG_SND_SOC_WM8510 is not set
 # CONFIG_SND_SOC_WM8523 is not set
@@ -6966,7 +6948,6 @@
 # CONFIG_SND_SOC_WM8974 is not set
 # CONFIG_SND_SOC_WM8978 is not set
 # CONFIG_SND_SOC_WM8985 is not set
-# CONFIG_SND_SOC_WSA881X is not set
 CONFIG_SND_SOC_ZL38060=m
 CONFIG_SND_SOC_MAX9759=m
 # CONFIG_SND_SOC_MT6351 is not set
@@ -8284,15 +8265,7 @@
 # CONFIG_RPMSG_VIRTIO is not set
 # end of Rpmsg drivers
 
-CONFIG_SOUNDWIRE=m
-
-#
-# SoundWire Devices
-#
-CONFIG_SOUNDWIRE_CADENCE=m
-CONFIG_SOUNDWIRE_INTEL=m
-# CONFIG_SOUNDWIRE_QCOM is not set
-CONFIG_SOUNDWIRE_GENERIC_ALLOCATION=m
+# CONFIG_SOUNDWIRE is not set
 
 #
 # SOC (System On Chip) specific Drivers
